core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-store/dot-pages.store.spec.ts

```diff
@@ -759,4 +761,34 @@ describe('DotPageStore', () => {
             inode: testInode
         });
     });
+
+    it('should handle error when a Workflow Action Request fails', (done) => {
+        const actions = [mockPublishAction];
+        const page = dotcmsContentletMock;
+        const error: HttpErrorResponse = new HttpErrorResponse({
+            error: {
+                message: 'Workflow Action is not available for current step'
```

Comment:
I believe it should be: Workflow Action is not available for the current step.?

Reply:
That's just a mock but I can change it